lib/phoenix/endpoint/adapter.ex

```diff
@@ -9,7 +9,10 @@ defmodule Phoenix.Endpoint.Adapter do
   """
   def start_link(otp_app, mod) do
     import Supervisor.Spec
-    pub_conf = config(otp_app, mod)[:pubsub]
+    conf       = config(otp_app, mod)
+    pub_conf   = conf[:pubsub]
+    asset_conf = conf[:assets]
```

Comment:
It is weird to call this assets because we don't call them assets anywhere (remember the brunch discussion?). Maybe it should be static?